dbms/src/Flash/Coprocessor/FilterConditions.cpp

```diff
@@ -20,11 +20,13 @@ namespace DB
 {
 FilterConditions::FilterConditions(
     const String & executor_id_,
-    const google::protobuf::RepeatedPtrField<tipb::Expr> & conditions_)
+    const google::protobuf::RepeatedPtrField<tipb::Expr> & conditions_,
+    const bool has_rf_conditions_)
```

Comment:
A field is needed to distinguish RF from normal condition. However, since we currently do not support adding filter operators, I will remove this attribute firstly.